pkg/tfbridge/info.go

```diff
@@ -128,6 +128,11 @@ type ProviderInfo struct {
 	//
 	// See also: pulumi/pulumi-terraform-bridge#1448
 	SkipValidateProviderConfigForPluginFramework bool
+
+	// Disables using detailed diff to determine diff changes and falls back on the length of TF Diff Attributes.
+	//
+	// See https://github.com/pulumi/pulumi-terraform-bridge/issues/1501
+	SkipDetailedDiffForChanges bool
```

Comment:
I don't think we want to support this feature until v4.0.0. When we close #1501 for real, we will want to remove this flag.
```suggestion
	// Disables using detailed diff to determine diff changes and falls back on the length of TF Diff Attributes.
	//
	// See https://github.com/pulumi/pulumi-terraform-bridge/issues/1501
	//
	// This flag is experimental and may be removed in a future bridge release.
	XSkipDetailedDiffForChanges bool
```